docs/source/explanations/compression.md

```diff
@@ -50,7 +50,7 @@ entries in this table above later ones.
 
 | Method                                                           | Accept-Encoding | Required Python Package |
 | ---------------------------------------------------------------- | --------------- | ----------------------- |
-| [blosc](https://www.blosc.org/)                                  | `blosc`         | `blosc`                 |
+| [blosc2](https://www.blosc.org/)                                 | `blosc2`        | `blosc2`                |
 | [lz4](https://en.wikipedia.org/wiki/LZ4_(compression_algorithm)) | `lz4`           | `lz4`                   |
 | [Zstandard](https://facebook.github.io/zstd/)                    | `zstd`          | `zstandard`             |
 | [gzip](https://en.wikipedia.org/wiki/Gzip)                       | `gzip`          | none (built in)         |
```

Comment:
```suggestion
| [gzip](https://en.wikipedia.org/wiki/Gzip)                       | `gzip`          | none (built in)         |
```

The line below this references `blosc` rathern than `blosc2`.  Should this get updated also?

```text
The Tiled Python *client* currently supports gzip and blosc (if the Python
package `blosc` is installed).
```

Reply:
Done!